specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -21879,13 +21879,13 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/workflowConfigurations/{configurationName}": {
-      "get": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/listWorkflowsConnections": {
+      "post": {
```

Comment:
why do you change to POST?
Looks like the existing list api is wrong, should just remove the resource name {configurationName}

Reply:
We had previously been denied moving to that route, since routes to plural words must contain ARM collection objects. This endpoint contains a single object response with multiple resources contained within. For clarity we would like to keep the route name plural to make it more clear what the route does, but we cannot change the data returned, as this would be a breaking change. 

The best compromise available to use was to change the route from GET to POST.

Reply:
This is breaking change, needs to be in a new api-version

Reply:
This is a new API route introduced in this version, 2022-09-01. We are changing the verb from the previous version within this release branch.